dotnet/CoreLib/Handlers/TextExtractionHandler.cs

```diff
@@ -34,10 +36,12 @@ public class TextExtractionHandler : IPipelineStepHandler
     public TextExtractionHandler(
         string stepName,
         IPipelineOrchestrator orchestrator,
+        IOcrEngine? ocrEngine = null,
```

Comment:
what happens if this is null, e.g. when using the serverless memory? does it mean that we support OCR only when running the service?

Reply:
Like this?  (Updated)

`.WithAzureFormRecognizer(Env.Var("AFR_ENDPOINT"), Env.Var("AFR_API_KEY"))`

Reply:
will fix in following PR